src/Console/Database/FixHtmlEncodingCommand.php

```diff
@@ -0,0 +1,300 @@
+     */
+    const ERROR_UPDATE_FAILED = 4;
+
+    /**
+     * Error code returned when rollback file cound not be created
+     *
+     * @var integer
+     */
+    const ERROR_ROLLBACK_FILE_FAILED = 5;
+
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('glpi:database:fix_html_encoding');
+        $this->setAliases(['db:fix_html']);
+        $this->setDescription(__('Fix Html encoding in database.'));
+
+        $this->addOption(
+            'itemtype',
+            null,
+            InputOption::VALUE_REQUIRED,
+            __('Itemtype to fix')
+        );
+
+        $this->addOption(
+            'id',
+            null,
+            InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
+            __('Id of the item to fix')
+        );
+
+        $this->addOption(
+            'field',
+            null,
+            InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
+            __('Field of the item to fix')
+        );
+
+        $this->addOption(
+            'dump',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            __('Path of file containing dump of existing values.')
+        );
+
+        $this->addUsage('--itemtype=ITILFollowup --id=42 --field=content [--dump=file_path.sql]');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        global $DB;
+
+        $this->checkArguments();
+
+        $itemtype = $input->getOption('itemtype');
+        $item_ids = $input->getOption('id');
+        $fields = $input->getOption('field');
+
+        if ($input->getOption('dump')) {
+            $this->dumpObjects();
+        }
+
+        $failed_items = [];
+        foreach ($item_ids as $item_id) {
+            $item = new $itemtype();
+            $item->getFromDB($item_id);
+
+            // update the item
+            $update = [];
+            foreach ($fields as $field) {
+                $update[$field] = $this->doubleEncoding($item->fields[$field]);
+                $update[$field] = $DB->escape($update[$field]);
+            }
+
+            $success = $DB->update(
+                $itemtype::getTable(),
+                $update,
+                ['id' => $item->fields['id']],
+            );
+            if (!$success) {
+                $failed_items[] = $item_id;
+                continue;
+            }
+        }
+
+        if (count($failed_items) > 0) {
+            $this->output->writeln(
+                '<error>' . sprintf(__('Unable to update %s items'), count($failed_items)) . '</error>',
+                OutputInterface::VERBOSITY_QUIET
+            );
+            foreach ($failed_items as $item_id) {
+                $this->output->writeln(
+                    '<error>' . sprintf(__('Itemtype %s ID %s'), $itemtype, $item_id) . '</error>',
+                    OutputInterface::VERBOSITY_QUIET
+                );
+            }
+            return self::ERROR_UPDATE_FAILED;
+        }
+
```

Comment:
You should add a success message, like:
```suggestion

        $output->writeln('<info>' . __('HTML encoding has been fixed.') . '</info>');
```